### ISSUE TITLE
prompt_toolkit breaks other applications

### ISSUE THREAD

**Issue by J535D165** [msdslab/asreview#130]
The fixed version in prompt_toolkkit breaks many applications. I think this issue has high prio. 

```
jonathans-mbp:recordlinkage jonathan$ ipython
Traceback (most recent call last):
  File "/Users/jonathan/.pyenv/versions/anaconda3-2018.12/bin/ipython", line 7, in <module>
    from IPython import start_ipython
  File "/Users/jonathan/.pyenv/versions/anaconda3-2018.12/lib/python3.7/site-packages/IPython/__init__.py", line 55, in <module>
    from .terminal.embed import embed
  File "/Users/jonathan/.pyenv/versions/anaconda3-2018.12/lib/python3.7/site-packages/IPython/terminal/embed.py", line 16, in <module>
    from IPython.terminal.interactiveshell import TerminalInteractiveShell
  File "/Users/jonathan/.pyenv/versions/anaconda3-2018.12/lib/python3.7/site-packages/IPython/terminal/interactiveshell.py", line 20, in <module>
    from prompt_toolkit.formatted_text import PygmentsTokens
ModuleNotFoundError: No module named 'prompt_toolkit.formatted_text'
jonathans-mbp:recordlinkage jonathan$ ipython
Traceback (most recent call last):
  File "/Users/jonathan/.pyenv/versions/anaconda3-2018.12/bin/ipython", line 7, in <module>
    from IPython import start_ipython
  File "/Users/jonathan/.pyenv/versions/anaconda3-2018.12/lib/python3.7/site-packages/IPython/__init__.py", line 55, in <module>
    from .terminal.embed import embed
  File "/Users/jonathan/.pyenv/versions/anaconda3-2018.12/lib/python3.7/site-packages/IPython/terminal/embed.py", line 16, in <module>
    from IPython.terminal.interactiveshell import TerminalInteractiveShell
  File "/Users/jonathan/.pyenv/versions/anaconda3-2018.12/lib/python3.7/site-packages/IPython/terminal/interactiveshell.py", line 20, in <module>
    from prompt_toolkit.formatted_text import PygmentsTokens
ModuleNotFoundError: No module named 'prompt_toolkit.formatted_text'

```

**Comment by J535D165** [comment]
See #153

**Comment by J535D165** [comment]
Fixed in 0.5.0. Thanks @qubixes
